ui/src/helpers/player-helper.js

```diff
@@ -82,6 +83,22 @@ const PlayerHelper = function() {
     }
   });
 
+  /**
+   * Checks if playerjs is used for preview or in editor
+   * @return {boolean}
+   */
+  this.isOnPreview = () => {
```

Comment:
Sorry but I am not clear why we've changed this. The new method (using window.parent) won't work in webOS/Tizen because there will be a CORS violation between us and the main application (the window.parent), and in android there is no window.parent because we aren't in any frame at all.

The way we had it previously relied on the URL params, which seems to make sense, but perhaps I have the wrong idea?

Reply:
The reason for changing this is that `isPreview` in these lines always evaluates to **true** even when the iframe is in the editor
```
const urlParams = new URLSearchParams(window.location.search);
const isPreview = urlParams.get('preview') === '1';
```

I have used `window.parent` since the structure of previewing the layout from the editor and on a separate window using the preview link, is the same. So, when we are on the editor and the layout is not being previewed, `isOnPreview()` is evaluated to **false**.

I was not aware that `window.parent` is not working for webOS/Tizen and android.

Maybe there's another way to identify when the layout is being previewed or not.

Reply:
Ah OK, I think we will need to add an `isEditor=1` parameter to these iframes when they are loaded in the editor - Mauro has an idea for how to do this